### PR TITLE
feat(SectionHeaderBacklink): add back for section header

### LINKED ISSUE #350: Add async header for automation's trigger pages.
Add async header for automation's trigger pages, i.e. header shows a questionnaire's name, a dropdown with delete and pause actions and the deactivation button.

image

### PR DESCRIPTION
**Pull request by zouxuoz**
resolve #350

### CODE REVIEW

File: client/components/SectionHeader/assets/back.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg fill="#000000" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```
Comment (radiansz) [inline client/components/SectionHeader/assets/back.svg]: May be name this file 'arrowLeft'? Back is so not clear what it is

Comment (zouxuoz) [inline client/components/SectionHeader/assets/back.svg]: fixed

**Comment by radiansz** [comment]
LGTM

**Comment by zouxuoz** [comment]
LGTM

File: client/components/SectionHeader/SectionHeaderBacklink.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <div {...element('arrow')} dangerouslySetInnerHTML={{ __html: arrowLeft }} />
+    <div {...element('label')}>
+      {label}
+    </div>
+  </Link>
+);
+
+SectionHeaderBacklink.propTypes = {
+  to: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
```
Comment (rosendi) [inline client/components/SectionHeader/SectionHeaderBacklink.js]: Missing proptypes...

```
block: PropTypes.func.isRequired,
elem: PropTypes.func.isRequired,
```

Comment (zouxuoz) [inline client/components/SectionHeader/SectionHeaderBacklink.js]: fixed

File: client/components/SectionHeader/SectionHeaderBacklink.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/no-danger */
+
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import withBEM from 'containers/withBEM';
+import arrowLeft from './assets/arrowLeft.svg';
```
Comment (rosendi) [inline client/components/SectionHeader/SectionHeaderBacklink.js]: ArrowLeft ArrowLeft.svg

Comment (zouxuoz) [inline client/components/SectionHeader/SectionHeaderBacklink.js]: fixed

**Comment by zouxuoz** [comment]
resolve #350

**Pull request merged**